proofs/alf/cvc5/programs/Arith.smt3

```diff
@@ -85,10 +85,10 @@
 ;; =============== for ARITH_POLY_NORM
 
 ; Definitions of monomials and polynomials.
-; A monomial is a list of terms that are ordered by `$compare_var` and a rational coefficient.
+; A monomial is a list of terms that are ordered by `$compare_var` and a coefficient.
 ; A polynomial is a list of monomials whose monomials are ordered by `$compare_var`.
 (declare-type @Monomial ())
-(declare-const @mon (-> (! Type :var T :implicit) T Real @Monomial))
+(declare-const @mon (-> (! Type :var T :implicit) T T @Monomial))
```

Comment:
I don't think we should do this change.

The problem here is that normalization must be done on integers when considering bitvectors, since normalization does not take into account overflows.

Using reals/rationals can be seen as a generalization.

Seen another way: the internal `PolyNorm` data structure in the cvc5 utils uses the `Rational` class.